splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/TestComponent.jsx

```diff
@@ -21,12 +20,13 @@ class TestComponent extends Component {
     }
 
     componentDidMount() {
-        console.log('getUnifiedConfigs: ', getUnifiedConfigs());
         axiosCallWrapper(this.props.serviceName)
             .then((response) => {
+                // eslint-disable-next-line no-console
                 console.log(response.data);
             })
             .catch((error) => {
+                // eslint-disable-next-line no-console
                 console.log(error);
             });
```

Comment:
Why we have a console here?

Reply:
It seems that, this component has no use

Reply:
Yes we can remove this component and its tests. It was kept from initial commit of template.

Reply:
removed